SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -2315,25 +2303,24 @@ USpatialActorChannel* USpatialNetDriver::CreateSpatialActorChannel(AActor* Actor
 	check(NetConnection != nullptr);
 
 	USpatialActorChannel* Channel = nullptr;
-	if (Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_Singleton))
+
+	if (Actor->GetClass()->HasAnySpatialClassFlags(SPATIALCLASS_Singleton) && !SpatialGDK::Singleton::ShouldHaveLocalWorkerAuthority(Actor, LoadBalanceStrategy, StaticComponentView))
 	{
-		Channel = GlobalStateManager->AddSingleton(Actor);
+		return Channel;
```

Comment:
Hmm, we still need a channel if you're non-auth. And doesn't CreateSpatialActorChannel get called in that instance?

Reply:
I think GitHub has misdirected this comment